autodns.py

```
@@ -102,37 +127,56 @@ def update_dns_web():
+    new_ip = get_client_ip()
 
-    headers = {"Authorization": f"Bearer {CF_API_TOKEN}", "Content-Type": "application/json"}
+    headers = {
+        "Authorization": f"Bearer {CF_API_TOKEN}",
+        "Content-Type": "application/json",
+    }
     dns_record = mapping[guid]["subdomain"]
     data = {"type": "A", "name": dns_record, "content": new_ip, "ttl": 1}
 
-    response = requests.get(f"{CF_API_URL_BASE}?name={dns_record}&type=A", headers=headers)
+    response = requests.get(
```

Ruff (0.15.4)

[error] Probable use of `requests` call without timeout

(S113)